src/App.tsx

```diff
@@ -1,7 +1,25 @@
-import React, { FC } from "react";
+import { FC } from "react";
+import { Route, Routes } from "react-router-dom";
+import Home from "./pages/Home";
+import SignIn from "./pages/Auth/SignIn";
+import SignUp from "./pages/Auth/SignUp";
+import NotFound from "./pages/NotFound";
+import Reset from "./pages/Auth/Reset";
+import { ROUTES } from "./utils/types";
+import ProtectedRoute from "./components/PrivateRoute";
 
 const App: FC = () => {
-  return <div>start</div>;
+  return (
+    <Routes>
+      <Route element={<ProtectedRoute />}>
+        <Route path={ROUTES.Home} element={<Home />} />
+      </Route>
+      <Route path={ROUTES.SIGN_IN} element={<SignIn />} />
+      <Route path={ROUTES.SIGN_UP} element={<SignUp />} />
+      <Route path={ROUTES.RESET} element={<Reset />} />
+      <Route path={ROUTES.NotFound} element={<NotFound />} />
+    </Routes>
```

Comment:
Check the routes logic. We need to redirect a user if he trying to enter in secured route. We need Private + Public routes for that.